sound/soc/sof/ipc.c

```diff
@@ -308,21 +308,20 @@ int sof_ipc_tx_message(struct snd_sof_ipc *ipc, u32 header,
 EXPORT_SYMBOL(sof_ipc_tx_message);
 
 /* handle reply message from DSP */
-int snd_sof_ipc_reply(struct snd_sof_dev *sdev, u32 msg_id)
+void snd_sof_ipc_reply(struct snd_sof_dev *sdev, u32 msg_id)
 {
 	struct snd_sof_ipc_msg *msg = &sdev->ipc->msg;
 
 	if (msg->ipc_complete) {
-		dev_err(sdev->dev, "error: no reply expected, received 0x%x",
+		dev_dbg(sdev->dev,
+			"no reply expected, received 0x%x, will be ignored",
 			msg_id);
-		return -EINVAL;
+		return;
```

Comment:
can you clarify what you are trying to fix?
Is this that the error level is too high?
This doesn't change any functionality so why change?

Reply:
@plbossart yes it isnt really an error. We see this every time the FW boots when the DSP resumes and it is misleading

Reply:
> then let's describe what the change really is in the commit message...

@plbossart i've updated the commit message now. Is this better?

Reply:
@ranj063 I find the updated commit still confusing, specifically this part "In fact, the host just ignores this message
and returns an error in such cases. But, the error message could be
misleading when trying to debug other issues."

It's odd to mention an 'error in such cases' when it's removed here? Something's not quite right.